src/index.js

```diff
@@ -98,4 +101,17 @@ app.get('/deletePledge/:pledgeId', async ({ params: { pledgeId } }, res) => {
   }
 });
 
+async function findNewNotifications() {
+  // notify for pledges that have expired
+  const expiredPledges = await db.findAllPledgesExpiredToNotify();
+  expiredPledges.map(async p => {
+    await postOnSlackMultipleChannels({
```

Comment:
this `await` is not working: `postOnSlackMultipleChannels` returns an array of promises and `await` only waits for promises. If you want to wait for every promise inside the array to be complete you have to use `Pomise.all` which wraps the array with another promise wwhich completes only when every single promise inside the array is complete.

```js
const postOnSlackMultipleChannels =
  (json, channels) => Promise.all(channels.map(c => postOnSlack({ channel: c, ...json })));
```